assets/css/main.css

```diff
@@ -291,3 +291,19 @@ content: 'Markdown';
 background: royalblue;
 color: whitesmoke ;
 }
+
+/* table */
+table {
+  border-spacing: 0;
+  border-collapse: collapse;
+}
+
+table th{
+  padding: 6px 13px;
+  border: 1px solid #dfe2e5;
```

Comment:
```suggestion
  border: 1px solid #dfe2e5;
  font-size: large;
```

Reply:
We could maybe Increase the Font Size of the Table Header for a bit of emaphasis? What's your takeon it?

Reply:
what size suitable for this?